app/components/Form.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import JSONSchemaForm from "react-jsonschema-form";
+import "bootstrap/dist/css/bootstrap.css";
```

Comment:
Why are we importing Bootstrap here? Let's get rid of this and style our form w/ CSS modules to fit the rest of the website's theme.

If you pay attention loading in this CSS override has strange effects across the site once this component mounts (underline hovers in the navigation, off-brand fonts, etc).